src/Symfony/Component/Validator/Constraints/Valid.php

```diff
@@ -43,4 +43,11 @@ public function addImplicitGroupName(string $group)
             parent::addImplicitGroupName($group);
         }
     }
+
+    public function __construct(array $options = null, bool $traverse = null, array $groups = null, $payload = null)
+    {
+        parent::__construct($options ?? [], $groups, $payload);
+
+        $this->traverse = $traverse ?? $this->traverse;
```

Comment:
I don't see the point of such check, what about `bool $traverse = true` and a simple assignation here?

Reply:
`traverse` might be assigned through `$options` too